Web/Design-Pattern-2/Assignment-3/src/GameInterface.java

```diff
@@ -0,0 +1,30 @@
+import java.util.Scanner;
+
+public class GameInterface {
+
+	public static void main(String[] args)
+	{
+		System.out.println("Enter 50 Overs For OneDay\n Enter 20 Overs For Twenty Twenty \n Enter 450 Overs For Test");
+		System.out.println("Enter Total No of Overs");
+		int overs = new Scanner(System.in).nextInt();
+		System.out.println("enter No of Teams");
+		int noOfTeams=new Scanner(System.in).nextInt();
+		System.out.println("Enter No Of players");
+		int noOfPlayers=new Scanner(System.in).nextInt();
+		
+		if (overs == 50) {
```

Comment:
ACTION: you should check all these conditions in initialize method of game class and initialize appropriate constructor of extended class.